docs/source/development/traffic_ops_api/v12/deliveryservice.rst

```diff
@@ -168,6 +168,8 @@ Delivery Service
   +--------------------------+--------+--------------------------------------------------------------------------------------------------------------------------------------+
   | ``sslKeyVersion``        |  int   |                                                                                                                                      |
   +--------------------------+--------+--------------------------------------------------------------------------------------------------------------------------------------+
+  | ``tenantId``             | int    | Owning tenant ID                                                                                                                     |
```

Comment:
Yes. I plan to do so in a separate PR after Ori's PR for adding it to the user is in.